Makefile

```diff
@@ -3,20 +3,20 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing
 develop:
-	pip install -r requirements-dev.txt
```

Comment:
@yohanboniface I initially asked the same question: in a virtualenv made for Python3, `bin/python` is `bin/python3` and therefore the change is unneeded.

But if it's unneeded in a virtualenv, it is also entirely harmless: in a virtualenv, `bin/python3` is `bin/python`, and therefore this change makes no difference.

However, this change makes it possible to run ideascube outside a virtualenv.

So I figured, if it makes no difference in a virtualenv, but it makes things better outside of a virtualenv, then why not? :smiley: